src/non_zero.rs

```diff
@@ -64,6 +68,41 @@ where
     }
 }
 
+impl<const LIMBS: usize> NonZero<UInt<LIMBS>>
+where
+    UInt<LIMBS>: Integer,
+{
+    /// Create a [`NonZero<UInt>`] from a [`NonZeroU8`] (const-friendly)
+    // TODO(tarcieri): replace with `const impl From<NonZeroU8>` when stable
```

Comment:
For now you can add non-`const` impls of these as well

Reply:
I left the TODOs because of `const`. Should I remove them?
Done otherwise.

Reply:
TODOs are good. Eventually this can be unified as a `const impl`.